### PR TITLE
Datetime fields in changeform

### PR DESCRIPTION
**Pull request by TysonRV**
- Giving some color to datetime fields in change_form

Far from perfect... but at least as a first version of it. 

Screenshot 2020-05-28 at 21 23 47

Screenshot 2020-05-28 at 21 30 03

### CODE REVIEW

**Comment by farridav** [comment]
Needs a rebase ;) make sure your not doing things that are already done , see https://github.com/farridav/django-jazzmin/pull/16 for up to date list of things that need sorting

File: jazzmin/static/jazzmin/js/change_form.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            '</div></p>'
+        )
+    });
+}
+
+function jazzIcons() {
+    setTimeout(function(){
+        $(".date-icon").attr("class", "fa fa-calendar");
+        $(".clock-icon").attr("class", "fa fa-clock");
+    }, 500);
```
Comment (TysonRV) [inline jazzmin/static/jazzmin/js/change_form.js:28]: This is mostly me testing the icons, but that's doable with CSS

Comment (farridav) [reply jazzmin/static/jazzmin/js/change_form.js:28]: SetTimeout .. that's nasty

Comment (TysonRV) [reply jazzmin/static/jazzmin/js/change_form.js:28]: Ha! yeah, we don't want that...

**Comment by farridav** [comment]
Are you happy with the final implementation ?

**Comment by TysonRV** [comment]
@farridav only for now. I would rather move on to something else and get this improved as a further step or second version. We still have to deal with making the whole thing responsive, so I'm sure more ideas will flow in :) 

Also, the most critical point are those timeouts.. but I haven't found out how to get rid of them and get the implementation I wanted without getting all full js-mode. Open to suggestions!

**Pull request merged**